keycloak/test-website/index.html

```diff
@@ -0,0 +1,243 @@
+<!doctype html>
```

Comment:
Is this file used for something?

Reply:
It can be used as tool to test keycloak, by runing it locally